<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>894</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Футбольные соревнования</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>871</width>
      <height>561</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>4</number>
    </property>
    <widget class="QWidget" name="tabAddTeam">
     <attribute name="title">
      <string>Добавить команду</string>
     </attribute>
     <widget class="QLineEdit" name="teamNameLineEdit">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Название команды</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="teamCityLineEdit">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>90</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Город</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="coachLastNameLineEdit">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>140</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Фамилия тренера</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="coachFirstNameLineEdit">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>190</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Имя тренера</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="coachMiddleNameLineEdit">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>240</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Отчество тренера</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="teamRankingLineEdit">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>290</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Место в рейтинге</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addTeamButton">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>340</y>
        <width>231</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавить команду</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabAddPlayer">
     <attribute name="title">
      <string>Добавить игрока</string>
     </attribute>
     <widget class="QLineEdit" name="playerLastNameLineEdit">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Фамилия игрока</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="playerFirstNameLineEdit">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>90</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Имя игрока</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="playerMiddleNameLineEdit">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>140</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Отчество игрока</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="playerAgeLineEdit">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>190</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Возраст игрока</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="playerPositionLineEdit">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>240</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Амплуа игрока</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="playerNumberLineEdit">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>290</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Номер игрока</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="teamForPlayerLineEdit">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>340</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Название команды</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addPlayerButton">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>390</y>
        <width>251</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавить игрока</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabAddStadium">
     <attribute name="title">
      <string>Добавить стадион</string>
     </attribute>
     <widget class="QLineEdit" name="stadiumNameLineEdit">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Название стадиона</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="stadiumCityLineEdit">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>90</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Город стадиона</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="stadiumCapacityLineEdit">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>140</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Вместимость стадиона</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="stadiumTicketPriceLineEdit">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>190</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Цена билета</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="teamForStadiumLineEdit">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>240</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Название команды</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addStadiumButton">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>290</y>
        <width>201</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавить стадион</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabAddMatch">
     <attribute name="title">
      <string>Добавить матч</string>
     </attribute>
     <widget class="QLineEdit" name="matchTeam1LineEdit">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Команда 1</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="matchTeam2LineEdit">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Команда 2</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="matchDateEdit">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>140</y>
        <width>281</width>
        <height>31</height>
       </rect>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="matchScoreLineEdit">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>180</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Счет матча</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="teamForMatchLineEdit">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>230</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Название команды</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addMatchButton">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>290</y>
        <width>211</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавить матч</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabQueries">
     <attribute name="title">
      <string>Запросы</string>
     </attribute>
     <widget class="QLineEdit" name="query1TeamLineEdit">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Название команды</string>
      </property>
     </widget>
     <widget class="QPushButton" name="query1Button">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>261</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Дата встречи и счет</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="query2StadiumLineEdit">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>140</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Название стадиона</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="query2DateEdit">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>190</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="query2Button">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>240</y>
        <width>261</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Поиск игрока</string>
      </property>
     </widget>
     <widget class="QPushButton" name="query3Button">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>310</y>
        <width>261</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Список игроков</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabTable">
     <attribute name="title">
      <string>Таблица</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>671</width>
        <height>451</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="refreshTableButton">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>10</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Обновить</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteRowButton">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>130</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Удалить</string>
      </property>
     </widget>
     <widget class="QPushButton" name="saveToFileButton">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>70</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Сохранить</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>894</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
